Install-Kite.md

```diff
@@ -3,6 +3,8 @@ layout: page
 title: Install the Kite Command Line Interface
 ---
 
+These are the instructions for downloading the Kite command line interface. The most up-to-date instructions for downloading the current version of the software are always available at [http://kitesdk.org/docs/current/Install-Kite.html](http://kitesdk.org/docs/current/Install-Kite.html).
```

Comment:
The reason for these revisions is that the version of CDH is almost always going to have an older version of the kite-dataset command. These are the instructions for installing and verifying that the latest version is in use. I'm not referring to any specific version, because that would have to be updated with each release. I added the "current" link up here in response to an earlier comment.